mlflow/tensorflow/__init__.py

```diff
@@ -753,6 +763,29 @@ def autolog(
         warnings.warn("Could not log to MLflow. TensorFlow versions below 1.12 are not supported.")
         return
 
+    def _extract_tensor_as_numpy_input_example_slice(next_input):
```

Comment:
Can we move these utility methods into a separate `mlflow/tensorflow/utils.py` module for readability?